recipe/meta.yaml

```
@@ -19,7 +19,7 @@ source:
 {% endif %}
 
 build:
-  number: {{ number }}
+  number: 0
```

Let's change this back to

```suggestion
  number: {{ number }}
```

It distinguishes the different ARM variants

cc @carterbox

---

Fixed in PR: https://github.com/conda-forge/cuda-nvcc-feedstock/pull/69